gateway/policy.go

```diff
@@ -17,6 +17,10 @@ import (
 	"github.com/TykTechnologies/tyk/user"
 )
 
+var (
```

Comment:
lets use `errors.go` for errors, no need to move existing ones yet, but we will eventually :)